__mocks__/got.js

```diff
@@ -0,0 +1,73 @@
+const people = require('./../assets/people.json');
+const films = require('./../assets/films.json');
+const species = require('./../assets/species.json');
+const planets = require('./../assets/planets.json');
+
+
+const got = (url, options) => {
+    let data, isError;
+    if (/^https:\/\/swapi.co\/api\/*/.test(url)) {
+
+        const path = url.replace('https://swapi.co/api/', '').split('/');
+        const pathNum = parseInt(path[1])-1;
+
+        if (isNaN(pathNum) || pathNum > 7) {
+            Promise.resolve({
+                data: {
+                    status: 400,
+                    error: "Bad request "
+                }
+            })
+        }
+
+        switch (path[0]) {
```

Comment:
`path[0]` does not at all describe what we're switching on. This would be confusing to anyone reading this later on